rclcpp/test/rclcpp/test_utilities.cpp

```diff
@@ -123,7 +123,6 @@ TEST(TestUtilities, test_context_basic_access_const_methods) {
   auto context1 = std::make_shared<const rclcpp::contexts::DefaultContext>();
 
   EXPECT_NE(nullptr, context1->get_init_options().get_rcl_init_options());
-  EXPECT_EQ(0u, context1->get_on_shutdown_callbacks().size());
```

Comment:
I have readded this line in https://github.com/ros2/rclcpp/pull/1639/commits/63244220c405a42916f4f74395f47515d5b92733, modified const qualifiers to make it work in https://github.com/ros2/rclcpp/pull/1639/commits/b27f448f7067e17a4aafa42e69af04003c652125.